src/components/Create/components/pages/utils/rules/inputIsValidUrlRule.ts

```diff
@@ -0,0 +1,30 @@
+import { t } from '@lingui/macro'
+import { RuleObject } from 'antd/lib/form'
+
+/**
+ * Rule to determine if the input already matches other `inputs`.
+ */
+export const inputIsValidUrlRule = (props: {
+  label?: string
+  prefix?: string
+  validateTrigger?: string
+}) => ({
+  validator: (rule: RuleObject, value: unknown) => {
+    if (value === undefined) return Promise.resolve()
+    if (typeof value !== 'string')
+      return Promise.reject(t`Invalid type - contact Juicebox Support`)
```

Comment:
**suggestion** Idk if contact Juicebox Support is an appropriate CTA. Is there any reason value wouldn't be a string, other than a bug?

Reply:
doesn't make sense for it to be anything other than a string - would just be a bug